packages/components/src/components/mobile-drawer/mobile-drawer-submenu-section.tsx

```diff
@@ -1,11 +1,15 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import Text from '../text/text';
 import Icon from '../icon/icon';
 
-const SubMenuSection = props => {
-    const { submenu_toggle_class, section_title, section_icon, children } = props;
+type TSubMenuSection = React.PropsWithChildren<{
+    submenu_toggle_class: string;
+    section_title: string | React.ReactElement;
```

Comment:
`section_title: React.ReactNode`

Perhaps try this and see if it works, ReactNode should also support text input.